apps/ca.c

```diff
@@ -246,7 +246,8 @@ int ca_main(int argc, char **argv)
     const char *serialfile = NULL, *subj = NULL;
     char *prog, *startdate = NULL, *enddate = NULL;
     char *dbfile = NULL, *f, *randfile = NULL;
-    char buf[3][BSIZE];
+    char new_cert[BSIZE] = { 0 };
```

Comment:
It is intentional, I prefer to fully erase the string.
It is not really necessary, as the first use of its , is 
 `            strcpy(new_cert, outdir); `